Aws/Aws.hs

```diff
@@ -52,6 +52,7 @@ import qualified Data.Text.Encoding           as T
 import qualified Data.Text.IO                 as T
 import qualified Network.HTTP.Conduit         as HTTP
 import           System.IO                    (stderr)
+import           Prelude
```

Comment:
The import of `Control.Applicative` is unnecessary with newer `base` versions, so moving the `Prelude` import silences the resulting warning in a backwards-compatible way. Edward Kmett introduced me to this trick.